stl.go

```diff
@@ -157,6 +157,9 @@ const (
 	stlTimecodeStatusIntendedForUse    = "1"
 )
 
+// TTI Special Extension Block Number
+const extensionBlockNumberReserverdUserData = 254
```

Comment:
There's a typo in the constant name, it should be `extensionBlockNumberReservedUserData`.

Also could you use the hexadecimal version of `254` instead ? (`0xfe`)

Reply:
done :)